### ISSUE TITLE
Anomaly with template attribute

### ISSUE THREAD

**Issue by maximedenes** [coq/coq#8696]
The following code:

```coq
#[template]
Inductive foo : Prop :=
  Foo : forall X : Prop, foo.
```

triggers

```
Error: Anomaly "Incorrect universe Prop declared for inductive type, inferred level is Set+1." Please report at http://coq.inria.fr/bugs/.
```

I'd expect a user error.

**Comment by SkySkimmer** [comment]
I guess we can just turn it into a user_error

**Comment by gares** [comment]
Which component raises it? Because user_errors can be catch by tactics...

**Comment by maximedenes** [comment]
> Which component raises it? Because user_errors can be catch by tactics...

The kernel.

**Comment by SkySkimmer** [comment]
That's a kernel error, the kernel raises anomalies if the universe 
levels of inductive types are incorrect.

Gaëtan Gilbert

**Comment by SkySkimmer** [comment]
Fixed probably around #9027
